test/integration/css/test/group-2.test.js

```diff
@@ -250,90 +250,6 @@ describe('CSS Support', () => {
     })
   })
 
-  describe('Basic Tailwind CSS', () => {
```

Comment:
These tests were added for `tailwindcss@1` and likely have not reflected the integration for a while, since Tailwind CSS is at v3 now. We could reintroduce these tests updated, in a separate PR instead.